lib/internal/modules/cjs/loader.js

```diff
@@ -22,8 +22,9 @@
 'use strict';
 
 const { NativeModule } = require('internal/bootstrap/loaders');
-const util = require('util');
+const { manifest } = require('internal/process/policy');
```

Comment:
I would prefer it this file was loaded only if policies are in use. Every module we add the longer it gets Node.js to start.

Reply:
I'm not convinced this has significant impact. Is there some metric we are trying to keep within? If startup time is more critical we could also take other approaches than avoiding loading code such as taking snapshots. Is this a nit or something that you wish to block on?

Reply:
+1 on code caches, but it's something that is kind of far in the future as far as I understand.

> Is there some metric we are trying to keep within?

@jasnell or @BridgeAR might have some number. However every file adds some millis to startup time.

Considering that it takes very little effort to load it at runtime, why not doing it?

Reply:
extra complexity from my point of view to do so for little gain from my understanding. even just bundling core into a single source text would have bigger savings than this I would think. the less complex the system via conditional branching the easier i find it to reason about and maintain.

Reply:
This PR gaves us 10%: https://github.com/nodejs/node/pull/20567. Most of those where added with the same reasoning, but as Node.js gets more feature, those pile up.

Reply:
I am definitely for lazy loading this! It is nothing that is required all the time. Adding snapshots is great and we want to do that on top of this. The current startup time is way to high and adding things as we used to doesn't make things any better. Instead we should always lazy load most files. This will also reduce the memory overhead for a simple Node.js application that does not use these features.

Reply:
lazy loaded now, rebased though